readme.md

```diff
@@ -98,16 +98,138 @@ Tooltips will now also load in from fixtures:
 * You can add tooltips to your fixture by adding a Tooltips.txt on [FixtureRoot]/[FixtureName]_tooltips/Tooltips.txt
 * You also need to add two executions to your maven dependency plugin to be able to load fixture tooltips:
 
-                    <execution>
-                        <id>unpack-dependencies</id>
-                        <phase>compile</phase>
-                        <goals>
-                            <goal>unpack-dependencies</goal>
-                        </goals>
-                        <configuration>
-                            <includes>**/Tooltips.txt</includes>
-                            <outputDirectory>${project.basedir}/wiki/TooltipData</outputDirectory>
-                        </configuration>
-                    </execution>
+        <execution>
+            <id>unpack-dependencies</id>
+            <phase>compile</phase>
+            <goals>
+                <goal>unpack-dependencies</goal>
+            </goals>
+            <configuration>
+                <includes>**/Tooltips.txt</includes>
+                <outputDirectory>${project.basedir}/wiki/TooltipData</outputDirectory>
+            </configuration>
+        </execution>
+             
                     
+# Getting Started with Bootstrap-plus FitNesse Theme
+This is meant for people who want to develop for the Bootstrap-plus FitNesse Theme.
+
+## Running the code 
+
+1. Put the correct Bootstrap plus version in the pom.xml.
+2. Press on "Maven" on the right side.
+    2. Open the folder “Lifecycle”.
+    2. Select “clean” and “install”. 
+    2. Press on “Run Maven Build”.
+
+## Code Set Up 
+
+### Pom file
+
+Bootstrap+ is a dependency of the Toolchain Plugin. 
+The Plugin is a maven project, so it is easier to make the Bootstrap+ a maven project too. 
+Every maven project has a pom file that contains the version number of the pom and the dependencies.  
+
+The most understandable way the keep track of your own version is to use a version with "-SNAPSHOT" that will ultimately be released. 
+For example if the most recent version is "2.0.5" then use "2.0.6-SNAPSHOT" for your local version.
+
+The Toolchain Plugin pom file also contains the Bootstrap+ Theme version. 
```

Comment:
Beetje raar verwoord. Bootstrap+ is a dependency of the Toolchain Plugin.

Reply:
We hebben de overbodige/incorrecte tekst er uit gehaald.